lib/form-layout.scss

```diff
@@ -90,6 +90,12 @@
     font-size: var(--pf-global--FontSize--sm);
   }
 
+  > .pf-c-form__helper-text {
```

Comment:
@garrett Here I add a fix for spacing for helper text in ct-form

Reply:
Why isn't it an alias for `.help-block` at https://github.com/skobyda/cockpit-certificates/blob/0a1a4d45a80601c3a476d651fd8e8df8d4b2ab48/lib/form-layout.scss#L290-L320 ?

Reply:
You mean doing something like this?
```
  > .pf-c-form__helper-text {
    @extend .help-block;
  }
```

Reply:
Fixed it